RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  ManageSubscriptionsView.swift
+//
+//
+//  Created by Andrés Boedo on 5/3/24.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct ManageSubscriptionsView: View {
+
+    @Environment(\.openURL)
+    var openURL
+
+    @StateObject
+    private var viewModel = ManageSubscriptionsViewModel()
+
+    init() { }
+
+    init(viewModel: ManageSubscriptionsViewModel) {
+        self._viewModel = .init(wrappedValue: viewModel)
+    }
+
+    var body: some View {
+        VStack {
+            HeaderView()
+
+            if let subscriptionInformation = self.viewModel.subscriptionInformation {
+                SubscriptionDetailsView(subscriptionInformation: subscriptionInformation,
+                                        refundRequestStatus: viewModel.refundRequestStatus)
+            }
+
+            Spacer()
+
+            ManageSubscriptionsButtonsView(viewModel: viewModel,
+                                           openURL: openURL)
+        }
+        .onAppear {
+            checkAndLoadSubscriptionInformation()
+        }
+    }
+
+}
+
+@available(iOS 15.0, *)
+private extension ManageSubscriptionsView {
+
+    func checkAndLoadSubscriptionInformation() {
+        if !viewModel.isLoaded {
+            Task {
+                await viewModel.loadSubscriptionInformation()
+            }
+        }
+    }
+
+}
+
+@available(iOS 15.0, *)
+struct HeaderView: View {
+    var body: some View {
+        Text("How can we help?")
```

Comment:
All these strings will be localized in the future

Reply:
the good thing about SwiftUI is that it'll pick up localization automatically here, we won't have to update lines like these with string literals as long as we have the values in String Catalogs

Reply:
ohhh nice